client/round.ts

```diff
@@ -122,11 +130,11 @@ export function roundView(model): VNode[] {
     ];
 }
 
-function playerInfo(model, color: string, rdiff: number | null) {
-    const username = model[color + "player"];
-    const title = model[color + "title"];
+function playerInfo(model: PyChessModel, color: string, rdiff: number | null) {
+    const username = model[color === "w"? "wplayer": "bplayer"];

```

Comment:
What's wrong with the old code?

Reply:
Error TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'PyChessModel'.
  No index signature with a parameter of type 'string' was found on type 'PyChessModel'.